libs/core/src/components/sage-link/sage-link.scss

```diff
@@ -0,0 +1,68 @@
+:host {
+  display: inline;
+
+  svg {
+    margin-left: 4px;
+  }
+}
+
+:host(:focus) {
+  border-radius: 6px;
+  outline: 4px solid #8ecafb;
+  outline-offset: 4px;
+}
```

Comment:
If we want to use `:host` thats ok with me but we should define some clear rules for when we use `:host` and when we use top level classes like `.sage-link`